owncloud/phoenix
# Fix token refresh flow

##  Description

- rename login button to login
- drop oidcSilentRedirect.vue 
- by default try to refresh the auth token - missing piece from some previous refactor

## How Has This Been Tested?
- setup phoenix with a true openid connect provider (e.g. kopano konnectd)
- wait for token expiry (life time depends on IdP)
- see in console that the token is refreshed

## Types of changes

## Checklist:



## Open tasks:
